src/index.test.ts

```diff
@@ -38,7 +38,7 @@ test('serverless-offline-watcher', async () => {
 
   // then... we have a mock watcher created
   expect(mockMakeWatcher).toHaveBeenCalledTimes(1);
-  expect(mockMakeWatcher).toHaveBeenCalledWith(serverlessMock.service.custom['serverless-offline-watcher']);
+  expect(mockMakeWatcher).toHaveBeenCalledWith(serverlessMock.service.custom['serverless-offline-watcher'], expect.any(Object));
```

Comment:
Optional: Not sure if this works, but think something like this would be neater:

```suggestion
  expect(mockMakeWatcher).toHaveBeenCalledWith(serverlessMock.service.custom['serverless-offline-watcher'], serverlessMock);
```